DataFormats/TrackReco/interface/HitPattern.h

```diff
@@ -445,6 +445,14 @@ class HitPattern
     int numberOfDTStationsWithRZView() const;
     int numberOfDTStationsWithBothViews() const;
 
+  //only used by ROOT IO rule to read v12 HitPatterns
+  static bool fillNewHitPatternWithOldHitPattern_v12(const uint16_t oldHitPattern[], 
```

Comment:
make this private?

Reply:
It can't be because it is called from an unknown function name inside the ROOT IO rule.